mlflow/pytorch/_lightning_autolog.py

```diff
@@ -489,13 +522,18 @@ def patched_fit(original, self, *args, **kwargs):
         if not any(isinstance(callbacks, __MLflowPLCallback) for callbacks in self.callbacks):
             self.callbacks += [
                 __MLflowPLCallback(
-                    client, metrics_logger, run_id, log_models, log_every_n_epoch, log_every_n_step
+                    client,
```

Comment:
Could we revert unrelated format change?

Reply:
I unfortunately don't have control over this as it's enforced by Ruff on commit

Reply:
@DouglasKrouth We pin ruff to `0.2.1` in this repo to avoid unrelated changes like this. You can use safe-without-formatting.